task/rhacs-m2m-authenticate/4.4.2/rhacs-m2m-exchange.yaml

```diff
@@ -0,0 +1,73 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: rhacs-m2m-exchange
+  labels:
+    apps.kubernetes.io/version: "4.5"
+  annotations:
+    tekton.dev/tags: security
+    tekton.dev/categories: Security
+    tekton.dev/displayName: "Exchange a service account token for a Red Hat Advanced Cluster Security short-lived token"
+    tekton.dev/platforms: "linux/amd64"
```

Comment:
Probably we will get this feedback from maintainers, but wondering what to fill in here since the roxctl image is a multi-arch manifest which supports more than just `linux/amd64` in theory, but let's see and feel free to ignore this one for the time being.